camelot/core.py

```diff
@@ -403,122 +433,36 @@ def set_edges(self, vertical, horizontal, joint_tol=2):
         for v in vertical:
             # find closest x coord
             # iterate over y coords and find closest start and end points
-            i = [
-                i
-                for i, t in enumerate(self.cols)
-                if np.isclose(v[0], t[0], atol=joint_tol)
-            ]
-            j = [
-                j
-                for j, t in enumerate(self.rows)
```

Comment:
By replacing this block with `generate_closest_points` the enumeration over `self.rows` is incorrectly changed to an enumeration over `self.cols`. The same is true for other replacments to `generate_closest_points`.

Reply:
I apologize for my inattention, the error has been corrected